docs/book/v2/getting-started/skeleton.md

```diff
@@ -549,7 +550,7 @@ You should see the message change as you go between the two URIs!
 
 You likely don't want to hardcode HTML into your middleware; so, let's use
 templates. This particular exercise assumes you chose to use the
-[Plates](http://platesphp.com) integration.
+[Plates](http://platesphp.com) integration. 
```

Comment:
Instead of pinning to plates, why not supply template examples for twig and zend-view? Expressive is all about choices right?